modules/server/src/main/ts/bedrock/core/Settings.ts

```diff
@@ -8,38 +8,23 @@ export interface BedrockSettings {
   readonly projectdir: string;
   readonly singleTimeout: number;
   readonly stopOnFailure: boolean;
-}
-
-export interface BedrockManualSettings extends BedrockSettings {
   readonly config: string;
   readonly coverage: string[];
   readonly customRoutes: string;
   readonly polyfills: string[];
   readonly testfiles: string[];
 }
 
+export type BedrockManualSettings = BedrockSettings;
```

Comment:
I'd like for this to remain the same as `Clis`, so can we either revert this change or move all the options from `forManual` to `commonOptions`?

Reply:
I've done the latter. I also removed the weird stopOnFailure__hidden setting. 

Can you please double-check the changes?